decidim-proposals/decidim-proposals.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
   s.add_dependency "decidim-comments", Decidim::Proposals.version
   s.add_dependency "decidim-core", Decidim::Proposals.version
   s.add_dependency "kaminari", "~> 1.0"
-  s.add_dependency "ransack", "~> 1.8"
+  s.add_dependency "ransack", "~> 1.8.7"
```

Comment:
Does this contribute to fixing the issue or is it collateral?

Reply:
is it collateral